modules/nw-egress-ips-about.adoc

```diff
@@ -20,7 +20,7 @@ To ensure that you can reliably allow access to the server from only that specif
 
 An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations.
 
-In some cluster configurations, application pods and ingress router pods run on the same node. If you configure an egress IP for an application project in this scenario, the IP is not used when you send a request to a route from the application project.
+In some cluster configurations, application pods and ingress router pods run on the same node. If you configure an egress IP for an application project in this scenario, the IP address is not used when you send a request to a route from the application project.
```

Comment:
A suggestion: I suggest changing 'egress IP' to 'egress IP address' for consistency, although our style guide lists using 'IP' as OK.

Reply:
@tmalove, EgressIPs are kind of weird, because the feature is actually called EgressIP or egress IP. I probably should change it to _EgressIP_ here perhaps.